addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRuleUnitTest.java

```diff
@@ -125,7 +126,7 @@ void shouldSendGetRequestWhenOriginalRequestWasNotGet() throws HttpMalformedHead
         // When
         rule.scan();
         // Then
-        assertEquals(1, httpMessagesSent.size());
+        assertThat(httpMessagesSent, hasSize(greaterThan(1)));
```

Comment:
Should this (and the others) be `greaterThan(0)`? That way we won't have to change it again if / when we move the custom payloads to `hidden_files.json`.

Reply:
Sure will tweak it.